.github/actions/setup-zeebe/action.yml

```diff
@@ -137,6 +137,7 @@ runs:
         -D aether.enhancedLocalRepository.splitRemote=true
         -D aether.syncContext.named.nameMapper=file-gav
         -D aether.syncContext.named.factory=file-lock
+        -D aether.syncContext.named.time=120
```

Comment:
:wrench: Ideally that would go into a separate PR where we can also make sure it's applied to all branches and not just 8.5

Reply:
I'll remove it for now even if it blocks the merge I guess

Reply:
yep

Reply:
my bad removed